doc/reading-list.md

```diff
@@ -79,6 +79,12 @@ RPC
 
 * AR of rpc layer
 
+Function Shipping
+---
+
+* Outdated document about how function shipping works in motr: [PDF](PDF/motr_function_shipping.pdf).
```

Comment:
I wouldn't say it's an outdated document. It's still quite up to date. Nothing was changed (in terms of Function Shipping) since the time it was written.

Reply:
@andriytk Do you like the idea to remove the "Outdated document about"?

Reply:
I'd put it as "Function Shipping usage" or something.